test/js-babel/main.js

```diff
@@ -0,0 +1,21 @@
+import Asciidoctor from 'asciidoctor.js'
+const asciidoctor = Asciidoctor()
+
+// BEFORE SUPPORT
+// Babel will move the import statement at the top of the file.
+// As a consequence, the reveal.js converter will be loaded before the initialization of Asciidoctor.js.
+//import 'asciidoctor-reveal.js'
+
+// NOW SUPPORTED
+import AsciidoctorRevealJs from 'asciidoctor-reveal.js'
```

Comment:
Just a nitpick but I think the project name is lowercase `reveal.js`, so the variable should be `AsciidoctorRevealjs` ?

Reply:
sure, why not